source/administration/config-settings.rst

```diff
@@ -1777,6 +1777,14 @@ So you don't miss messages, please make sure to change this value to an email yo
 | This feature's ``config.json`` setting is ``"FeedbackEmail": ""`` with string input.                                                                                 |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Notification Reply-To Address
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Email address used in the Reply-To header when sending notification emails from Mattermost..
```

Comment:
Wondering the use case for this setting? Is it primarily for Martin's email plugin?

Reply:
Yes that is why it was added. (Fixed the dbl period issue too)

Reply:
Wondering if the plugin would be ready to mention here as a sample use case? https://github.com/DSchalla/mailermost-plugin

Reply:
@mkraft - I'd defer that question to you :)

Reply:
I'll add a note to follow up on this afterwards.